apps/financial-aid/web-osk/src/routes/status/FileUpload/fileUpload.treat.ts

```diff
@@ -0,0 +1,11 @@
+import { theme } from '@island.is/island-ui/theme'
+import { style } from 'treat'
+
+export const widthAlmostFull = style({
```

Comment:
This is declared in a couple of places, not sure if we should re-use it or not, for now it is c/p